app/vmalert/provider/alertmanager.go

```diff
@@ -0,0 +1,11 @@
+package provider
+
+import "github.com/VictoriaMetrics/VictoriaMetrics/app/vmalert/config"
+
+// AlertManager represents integration provider with Prometheus alert manager
+type AlertManager struct{}
+
+// Fire fires an alert
+func (a *AlertManager) Fire(rule config.AlertRule) error {
```

Comment:
There should be `Resolved` as well. However, I'd suggest to have `Send` instead where you decide what action to take or what state to send.